Alan's Sandbox [DNM] [TM Only]

AlanParkerMasters opened the pull request:
# About the pull request
An amalgamation of code, built for spess people.
Do not merge obviously. Meant for whatever I am working on in a round.  
<!-- Remove this text and explain what the purpose of your PR is.

Mention if you have tested your changes. If you changed a map, make sure you used the mapmerge tool.
If this is an Issue Correction, you can type "Fixes Issue #169420" to link the PR to the corresponding Issue number #169420.

Remember: something that is self-evident to you might not be to others. Explain your rationale fully, even if you feel it goes without saying. -->

# Explain why it's good for the game
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>


# Changelog
:cl:
add: Added something
del: Removed old things
qol: made something easier to use
balance: rebalanced something
fix: fixed a few things
spellcheck: fixed a few typos
ui: changed something relating to user interfaces
code: changed some code
refactor: refactored some code
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
mapadd: added a new map or section to a map
maptweak: tweaked a map
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

AlanParkerMasters committed:
aa

AlanParkerMasters committed:
aa

AlanParkerMasters committed:
yup

AlanParkerMasters committed:
finally

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
det hat fixes/fedora

AlanParkerMasters committed:
ANOTHER hat fix

AlanParkerMasters committed:
even more hat fixes (guh)

AlanParkerMasters committed:
line art and fixes

AlanParkerMasters committed:
fixes (corpse spawner bug)

AlanParkerMasters committed:
corpse spawner removal bug

AlanParkerMasters committed:
more corpse spawner fixes

tyeagg committed:
liquids and submerge

tyeagg committed:
nooverlay

AlanParkerMasters committed:
adds two autodocs to Hybrisa hospital for PVP map

AlanParkerMasters committed:
Hybrisa repaired map complete

AlanParkerMasters committed:
hybrisa repaired map fix

tyeagg committed:
bettertrans

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
fixes planes issue (repaired Hybrisa)

AlanParkerMasters committed:
Hybrisa repaired map fix

AlanParkerMasters committed:
var edit fix (repaired)

AlanParkerMasters committed:
couple of varedited .dmi's removed

AlanParkerMasters committed:
proper spacecraft fuel tanks (repaired-hy)

AlanParkerMasters committed:
atm fixes and giant driller addition (clean-hy)

AlanParkerMasters committed:
more fixes

AlanParkerMasters committed:
banned neighbor tiles fixes (clean-hy)

AlanParkerMasters committed:
more banned neighbor fixes

AlanParkerMasters committed:
another neighbor fix

AlanParkerMasters committed:
even more neighbor fixes

AlanParkerMasters committed:
a

AlanParkerMasters committed:
aa

AlanParkerMasters committed:
dmi fix (c-h)

AlanParkerMasters committed:
twe airbase clean Hybrisa fix

AlanParkerMasters committed:
removes weeds from repaired

tyeagg committed:
Merge branch 'master' into submerge

tyeagg committed:
fish

tyeagg committed:
otogitypes

tyeagg committed:
ghostsdontshakegrass

AlanParkerMasters committed:
replaces Engi ship with expanded caverns (repaired-hy)

AlanParkerMasters committed:
replaces some unacid/explode proof shutters (rep-hy)

AlanParkerMasters committed:
more fixes (rep-hy)

AlanParkerMasters committed:
Adds and fixes proper phone lines (rep-hy)

AlanParkerMasters committed:
proper functioning ID doors + klaxon blaring removal

AlanParkerMasters committed:
check fix

AlanParkerMasters committed:
anchors and layers Hybrisa decals properly

AlanParkerMasters committed:
another decal fix

AlanParkerMasters committed:
makes Hybrisa dirt and trash decals cleanable

AlanParkerMasters committed:
Merge branch 'master' into hybrisa_port

AlanParkerMasters committed:
actually gives IDs accountant numbers

AlanParkerMasters committed:
test fix

AlanParkerMasters committed:
door ID fix

AlanParkerMasters committed:
area check fix

AlanParkerMasters committed:
adds working ambulances and cop cars

AlanParkerMasters committed:
aa

AlanParkerMasters committed:
random map fixes (clean hy)

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
bed.dm conflict fix

AlanParkerMasters committed:
corporate rifle back

AlanParkerMasters committed:
brings back pmc beret

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
removes random survivor weaponry by request

AlanParkerMasters committed:
check fixes for dmi file limits

workarounds for dmi file limit in uniform0.dmi by moving most Hybrisa uniforms to uniform1.dmi until icon reorganization

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
vehicle name fix, shutter dupe fix, container layer fix

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

tyeagg committed:
Merge branch 'master' into submerge

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
fix random obj preset

AlanParkerMasters committed:
MRE and tear gas fixes

AlanParkerMasters committed:
wy mre fixes

AlanParkerMasters committed:
sg armor fix

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
Merge branch 'master' into hybrisa_port

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

tyeagg committed:
Merge branch 'master' into submerge

tyeagg committed:
caliope

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
readds static HAI sentry

tyeagg committed:
Merge branch 'master' into submerge

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
sentry fix

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
ammo casings fix

AlanParkerMasters committed:
base NPCs and stuff

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into alan-sandbox-hybrisa-zombies-npcs

AlanParkerMasters committed:
Merge remote-tracking branch 'tyeagg/submerge' into alan-sandbox-hybrisa-zombies-npcs

AlanParkerMasters committed:
Update LV759_Hybrisa_Prospera_repaired.dmm

AlanParkerMasters committed:
Update LV759_Hybrisa_Prospera_repaired.dmm

AlanParkerMasters committed:
Update LV759_Hybrisa_Prospera_repaired.dmm

AlanParkerMasters committed:
Update LV759_Hybrisa_Prospera_repaired.dmm

cm13-github commented:
<!-- tgs_test_merge_comment -->
## Test merge deployment history:

<details><summary>Test Merge Deployed @ 07/01/2025 03:00:10 +00:00:</summary>

##### Server Instance
PVECM: Red

##### Merged By
alanparkermasters

##### Revision
Origin: 9a7b5d40e23eec6a2695f8e1acc5b7d677387e77
Pull Request: 919c8f1a8f76d3bb76b1b92daf7b91fea1a9e4f3
Server: 8de78d049081a491b0e22413e1d1dac1e0a39b35
</details>

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into alan-sandbox-hybrisa-zombies-npcs

AlanParkerMasters committed:
Update LV759_Hybrisa_Prospera_repaired.dmm

AlanParkerMasters committed:
weather & lmg adds

AlanParkerMasters committed:
Merge branch 'alan-sandbox-hybrisa-zombies-npcs-test3' of https://github.com/AlanParkerMasters/cmss13-pve into alan-sandbox

AlanParkerMasters committed:
fix for overlay bug

AlanParkerMasters committed:
fixes water gaps

AlanParkerMasters committed:
prop in room add

AlanParkerMasters committed:
landmark overhaul

reenables these landmarks, organizes them neatly, and makes use of equipment presets sitting in code

AlanParkerMasters committed:
pilot & unequipped USCM + organization

AlanParkerMasters committed:
dogtag fix

AlanParkerMasters committed:
HAI landmarks

AlanParkerMasters committed:
reverts icon change (not obvious enough), organizes UPP colonists, adds UPP sniper

AlanParkerMasters committed:
Merge branch 'HAI-landmarks' into alan-sandbox-hybrisa-zombies-updatedbranch

AlanParkerMasters committed:
Merge branch 'landmark-spawners' into alan-sandbox-hybrisa-zombies-updatedbranch

AlanParkerMasters committed:
move observer landmark

AlanParkerMasters committed:
crime scenes for players

AlanParkerMasters committed:
miner in caves

AlanParkerMasters committed:
Merge branch 'alan-sandbox-hybrisa-zombies-updatedbranch' into alan-sandbox

AlanParkerMasters committed:
add corpses to morgue

AlanParkerMasters committed:
updates
